node/hubController.js

```diff
@@ -3,15 +3,17 @@
 'use strict';
 
 var q = require('q');
-var hubsConfig = require("./hubsConfig");
+var HubsConfig = require("./hubsConfig");
 var OpenT2TError = require('opent2t').OpenT2TError;
 var OpenT2TConstants = require('opent2t').OpenT2TConstants;
+var OpenT2TLogger = require('opent2t').Logger;
 
 class HubController {
 
-    constructor() {
+    constructor(logLevel = "info") {
         this.supportedHubsCache = undefined;
         this.OpenT2T = require('opent2t').OpenT2T;
+        this.ConsoleLogger = new OpenT2TLogger(logLevel);
```

Comment:
why is this named ConsoleLogger? won't it be whatever opent2tlogger is configured for?